Car/src/activity/journey/journey-activity/create-journey/NewJourneyDetailsPage/NewJourneyDetailsPage.tsx

```diff
@@ -82,10 +82,17 @@ const NewJourneyDetailsPage = (props: NewJourneyDetailsPageProps) => {
                         type: value.stopType,
                         id: 0,
                         journeyId: 0,
-                        user: null
+                        user: null,
+                        userId: Number(user?.id)
                     };
                 })
-        });
+        };
+
+        await JourneyService.add(newJourney)
+            .then(() => {
+                Alert.alert("Ride successfully published");
+                navigation.navigate("Journey");
+            }).catch(() => Alert.alert("Error"));
```

Comment:
please add more information:
ex. "Ride publishing is failed"

Reply:
Done